resources/dicts/relationship_events/normal_interactions/platonic/decrease.json

```diff
@@ -1,194 +1,244 @@
-		"interactions": [
-			"m_c snarls at r_c for coming too close to the herbs.",
-			"m_c shoos r_c away, not having time for curious little kits.",
-			"m_c tells r_c if {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} stay away from the medicine cat den {PRONOUN/m_c/subject} will feed {PRONOUN/r_c/object} nasty tasting herbs!",
-			"r_c scrunches {PRONOUN/r_c/poss} nose at m_c and says {PRONOUN/m_c/subject} {VERB/m_c/smell/smells} weird."
-		],
-		"main_status_constraint": [
-			"medicine cat",
-			"medicine cat apprentice"
-		],
-		"main_trait_constraint": [
-			"cold",
-			"grumpy",
-			"fierce"
-		],
-		"random_status_constraint": [
-			"kitten"
-		],
-		"reaction_random_cat": {
-			"platonic": "decrease"
-		}
-	},
-	{
-		"id": "platonic_de_high4",
-		"intensity": "high",
-		"interactions": [
-			"r_c is caught tossing out a gift m_c <i>just</i> gave {PRONOUN/r_c/object}.",
-			"m_c caught r_c complaining about {PRONOUN/m_c/object} behind {PRONOUN/m_c/poss} back.",
-			"m_c interrupted r_c when {PRONOUN/r_c/subject} {VERB/r_c/were/was} very busy with an important task, ruining {PRONOUN/r_c/poss} focus.",
-			"r_c purposefully picked the smallest prey for m_c when {PRONOUN/m_c/subject} asked for help.",
-			"r_c was found spreading rumors about m_c stealing stuff from {PRONOUN/r_c/object}.",
-			"r_c was caught hiding m_c's belongings.",
-			"r_c publicly mocked m_c during a Gathering.",
-			"m_c discovered r_c was sabotaging {PRONOUN/m_c/poss} efforts at work.",
-			"m_c walked in on r_c destroying a memento {PRONOUN/m_c/subject} cherished."
-		]
-	}
-]
+[
+	{
+		"id": "platonic_de_low1",
+		"intensity": "low",
+		"interactions": [
+			"m_c thinks r_c isn't very considerate of others.",
+			"m_c gets a little irritated seeing r_c ignore a mess in camp.",
+			"r_c accidentally gave m_c a kind of prey {PRONOUN/m_c/subject} didn't like.",
+			"r_c doesn't seem very concerned when m_c has a cough.",
+			"m_c feels like r_c wasn't listening to {PRONOUN/m_c/object}.",
+			"r_c didn't put a lot of thought into {PRONOUN/r_c/poss} answer for m_c's question.",
+			"m_c doesn't see why r_c is upset about something trivial.",
+			"r_c yawns while m_c is trying to tell {PRONOUN/r_c/object} something.",
+			"r_c doesn't get upset about something that m_c thinks {PRONOUN/r_c/subject} should.",
+			"m_c asks for the last of r_c's food, but r_c says no.",
+			"r_c doesn't help m_c fix {PRONOUN/m_c/poss} nest."
+		]
+	},
+	{
+		"id": "platonic_de_low2",
+		"intensity": "low",
+		"interactions": [
+			"m_c and r_c have an argument about who should get a pretty feather, and end up destroying it in their fight.",
+			"m_c has drawn the ire of r_c by deliberately taking the last of {PRONOUN/r_c/poss} favorite nesting materials for {PRONOUN/m_c/self}.",
+			"m_c and r_c keep talking over each other on accident.",
+			"After m_c cleans up an area, r_c accidentally messes it up again.",
+			"m_c tells something to r_c, but realizes r_c can't relate at all.",

```

Comment:
Purrrsonally would prefer this to be "m_c tells something to r_c, but realizes {PRONOUN/r_c/subject} can't relate at all."